Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -380,6 +378,17 @@ theorem rightMoves_mul_cases {x y : PGame} (k) {P : (x * y).RightMoves → Prop}
   · apply hl
   · apply hr
 
+/-- `x * y` and `y * x` have the same moves. -/
+protected lemma mul_comm (x y : PGame) : x * y ≡ y * x :=
```

Comment:
Just a small note to maintainers: since the ultimate goal is to replace `Relabelling` in favor of `Identical`, this proof is preferrable to the more direct one that makes use of `mulCommRelabelling`.